tests/pretty_format_json_test.py

```diff
@@ -108,76 +107,51 @@ def test_badfile_main():
     assert ret == 1
 
 
-def test_diffing_output():
-    table_tests = [
-        {
-            'name': 'diff_test_1',
-            'source': """
-{
-    "key1": "val1",
-    "key2":   2,
-"array_key": [1, 2, 3],
-        "object":{
-        "bool_key": true
-      }
-}
-""",
-            'target': """
-{
-  "array_key": [
-    1,
-    2,
-    3
-  ],
-  "key1": "val1",
-  "key2": 2,
-  "object": {
-    "bool_key": true
+def test_diffing_output(capsys):
+    resource_path = get_resource_path('not_pretty_formatted_json.json')
+    expected_retval = 1
+    expected_diff = '''
```

Comment:
looks like this is printing twice

Reply:
Good catch! I had a weird print statement in `get_diff` function; fixed in 93b7b66